internal/service/federatedsettingsidentityprovider/data_source_federated_settings_identity_providers_test.go

```diff
@@ -28,12 +28,12 @@ func TestAccFederatedSettingsIdentityProvidersDS_basic(t *testing.T) {
 
 					resource.TestCheckResourceAttrSet(resourceName, "federation_settings_id"),
```

Comment:
Curious why this did not fail with the original PR that changed the request behaviour

Reply:
i suppose it's not defined the API response order